<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VocalFly</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/icon/icon/vocalflyicon.png</normaloff>:/icon/icon/vocalflyicon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>543</x>
      <y>611</y>
      <width>246</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Appliquer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>277</x>
      <y>611</y>
      <width>246</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Annuler</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>930</width>
      <height>250</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Choix du service de vocalisation :</string>
    </property>
    <widget class="QRadioButton" name="Google">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>761</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Cette voix est issue de Google,
la voix est plutôt jolie mais
elle nécessite internet pour fonctionner.</string>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Google_Speech (nécessite internet, la meilleure)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="EspeakNG">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>761</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Cette voix est 100% libre,
elle est aussi entièrement locale
(elle ne nécessite pas internet)
mais en revanche elle est très robotique !</string>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Espeak-ng (ne nécessite pas internet, voix moche)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="pico2wave">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>761</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Cette voix est 100% libre,
elle est aussi entièrement locale
(elle ne nécessite pas internet)
et elle est mieux que Espeak-NG !</string>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>pico2wave (ne nécessite pas internet, voix mieux)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="TestGoogle">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>50</y>
       <width>141</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Test voix
Google</string>
     </property>
    </widget>
    <widget class="QPushButton" name="TestSpeak">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>110</y>
       <width>141</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Test voix
Espeak</string>
     </property>
    </widget>
    <widget class="QPushButton" name="TestPico">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>170</y>
       <width>140</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Test voix
Pico</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>930</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Activation du service de vocalisation :</string>
    </property>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>761</width>
       <height>81</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Activation/Désactivation de VocalFly</string>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Activation de VocalFly</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>611</y>
      <width>246</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>791</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/icon/icon/VocalFly barre V3.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>29</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuVocalFly">
    <property name="title">
     <string>VocalFly</string>
    </property>
    <addaction name="actionFermer"/>
   </widget>
   <widget class="QMenu" name="menu_propos">
    <property name="title">
     <string>À propos...</string>
    </property>
    <addaction name="action_propos_de_VocalFly"/>
    <addaction name="action_propos_de_DVKbuntu"/>
    <addaction name="action_propos_d_HandyOpenSource"/>
   </widget>
   <addaction name="menuVocalFly"/>
   <addaction name="menu_propos"/>
  </widget>
  <action name="actionFermer">
   <property name="text">
    <string>Fermer</string>
   </property>
  </action>
  <action name="actionR_initialiser">
   <property name="text">
    <string>Réinitialiser</string>
   </property>
  </action>
  <action name="action_propos_de_VocalFly">
   <property name="text">
    <string>À propos de VocalFly</string>
   </property>
  </action>
  <action name="action_propos_de_DVKbuntu">
   <property name="text">
    <string>À propos de DVKbuntu</string>
   </property>
  </action>
  <action name="action_propos_d_HandyOpenSource">
   <property name="text">
    <string>À propos d'HandyOpenSource</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
